ui/src/app/views/concept-table/component.tsx

```diff
@@ -93,9 +110,16 @@ export class ConceptTable extends React.Component<{concepts: Concept[];
     super(props);
     this.state = {
       selectedConcepts: props.selectedConcepts,
-      selectedVocabularies: []
+      selectedVocabularies: [],
+      loading: false,
+      first: 0,
+      rows: 10,
+      totalRecords: props.concepts.length,
+      pageNumber: 0,
+      pageConcepts: props.concepts.slice(0, 10)
     };
     this.filterImageSrc = 'filter';
+    this.onPage = this.onPage.bind(this);
```

Comment:
Do you need to use bind, instead of binding the function below to `() => this.onPage()`? (I think we prefer to use lambdas over binds when we can)